docs/design/autoware-interfaces/ad-api/features/cooperation.md

```diff
@@ -0,0 +1,66 @@
+# Cooperation
+
+## Related API
+
+- {{ link_ad_api('/api/planning/velocity_factors') }}
+- {{ link_ad_api('/api/planning/steering_factors') }}
+- {{ link_ad_api('/api/planning/cooperation/set_commands') }}
+- {{ link_ad_api('/api/planning/cooperation/set_policies') }}
+- {{ link_ad_api('/api/planning/cooperation/get_policies') }}
+
+## Description
+
+Some planning modules can receive the operator decision and reflect it in their behavior.
+These modules have their own decisions, but their behavior is based on the operator decisions.
+The operator can check the module decision and change the operator decision if necessary.
+If the module supports this feature, [cooperation status](../types/autoware_adapi_v1_msgs/msg/CooperationStatus.md) is provided in [velocity factors or steering factors](./planning-factors.md).
+
+![cooperation-architecture](./cooperation/architecture.drawio.svg)
+
+## Scene ID
+
+The cooperation status contains an ID to distinguish scenes.
+This ID is generated for each scene that requires decision and doesn't change until the scene is completed or canceled.
+To set the operator decision, it needs to specify the ID by selecting the target scene from the cooperation status.
+In practice, the application can hides the specification of the ID and provides an abstracted interface to the operator.
+
+## Decisions
+
+The module decision is either deactivate or activate. The operator decision is also evaluated in one of these.
```

Comment:
There could a better phase other than "deactivate"/"activate".
It sounds like operator would terminate the module itself if they "deactivate" a module.

Alternative examples could be:
- approve / accept / apply module decision
- reject / ignore module decision

Reply:
Those words were originally considered as candidates, but they were not adopted because it had a different meaning. The equivalent of "approve / accept / apply" is "autonomous", and the equivalent of "reject / ignore" is "deactivate / activate".

This is because the operator decisions do not describe how the module decisions should be processed, but rather how the vehicle behavior should be determined.